docs/pbc/all/merchant-management/202407.0/marketplace/marketplace-merchant-commission-feature-overview.md

```diff
@@ -0,0 +1,192 @@
+
+* `category IS IN 'electronics';smart-watches' AND attribute.color IS IN 'black;blue'`
+* `category IS IN 'electronics';smart-watches' OR attribute.color IS IN 'black;blue'`
+
+When conditions are combined by the `OR` operator, they don't exclude each other. If an order fulfills both such conditions, the commission is still applied.
+
+
+
+### Merchant-based conditions
+
+
+The `merchants_allow_list` attribute lets you apply a commission to one or more particular merchants. This is useful when you only want to apply a commission to some of your merchants; or when you have a universal commission for most merchants and a special commission for some of them. For example, your setup could be as follows. We simplified the commissions for this example by removing irrelevant fields.
+
+| KEY | PRIORITY | AMOUNT | merchants_allow_list |
+| mc01 | 1 | 10 | MER000002,MER000004 |
+| mc02 | 2 | 5 |   |
+
+In the prior example, the system first checks if an order is fulfilled by merchants `MER000002` or `MER000004`. If this condition is fulfilled, the `mc01` commission is applied. If an order is fulfilled by any other merchant, the universal commission `mc02` is applied. The empty `merchants_allow_list` attribute means that this commission applies to all merchants.
+
+If you have multiple commissions, depending on your setup, you will have to use priorities and groups to make sure that relevant commissions are applied. In the prior example, the merchant-specific commission with priority `1` is validated before the universal commission. If the merchant-specific commission had a lower priority, the universal commission would be applied because it doesn't have a merchant condition. For more details about priority, see []
+
+
+
+
+## Merchant commission priority and groups
+
+Priority and groups are used to make sure relevant commissions are applied when there are multiple commissions in a system.
+
+Merchant commission groups are used when multiple commissions need to be applied to per order or order item. By default, there are `primary` and `secondary` commission groups. When commissions of both groups exist in a shop, for each order or order item, one commission from each group is applied; the commission from the primary group is applied first.
+
+Merchant commission priority is used to define which commission is to be applied within a commission group when there're multiple commissions in a system. Priority is defined in an ascending order starting from 1. If groups aren't used in a system, but there're multiple commissions, priority is used to define which commission is applied in the system.
+
+For example, your setup with groups and priority could be as follows. We simplified the commissions for this example by removing irrelevant fields.
+
+| Commission Key | Priority | Group |
+| - | - | - |
+| MC01 | 1 | Primary |
+| MC02 | 2 | Primary |
+| MC03 | 2 | Secondary |
+| MC04 | 1 | Secondary |
+
```

Comment:
Wrong table:

| MC01 | 1 | Primary |
| MC02 | 2 | Primary |
| MC03 | 1 | Secondary |
| MC04 | 2| Secondary |

Reply:
I updated the table to make the example more realistic